src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
-import cherrypy
+import Cookie
```

Comment:
The appears to break Nautilus on openSUSE Leap, because it uses Python 3. (I gather `import Cookie` only works on Python 2.)

Reply:
Yes, it seems the origin PR has been adapted for Nautilus. [Master](https://github.com/ceph/ceph/commit/3c72dc309936b23e413dc1aee8ca49c795c48a0f#diff-97f79e49e6979fdca7f121c9c4e1a3bc173163c54f7329722b6d95003f90354bR4) -> [Octopus](https://github.com/ceph/ceph/blob/octopus/src/pybind/mgr/dashboard/controllers/auth.py#L4) are using `http.cookies`.

Reply:
https://github.com/ceph/ceph/pull/40116 - fixed in this PR